packages/components/src/color-picker/index.native.js

```diff
@@ -89,7 +89,7 @@ function ColorPicker( {
 			return;
 		}
 		setColor( currentColor );
-	}, [ currentColor ] );
+	}, [ currentColor, setColor ] );
```

Comment:
What about this one? Should this be in a `ref`? I haven't tested but [this one](https://github.com/WordPress/gutenberg/blob/d3157c73eaf85455021712839b261975a7f9dc7a/packages/block-library/src/cover/edit.native.js#L371) could be problematic maybe.